public/content/developers/docs/transactions/index.md

```diff
@@ -160,13 +160,20 @@ The base fee will be burned **-0.00399 ETH**
 
 Validator keeps the tip **+0.000210 ETH**
 
-Gas is required for any smart contract interaction too.
 
 ![Diagram showing how unused gas is refunded](./gas-tx.png)
 _Diagram adapted from [Ethereum EVM illustrated](https://takenobu-hs.github.io/downloads/ethereum_evm_illustrated.pdf)_
 
 Any gas not used in a transaction is refunded to the user account.
 
+### Smart contract interactions {#smart-contract-interactions}
+
+Gas is required for any transaction that alters the state of a smart contract.
```

Comment:
This wording is a little redundant, as a "transaction" does commit alterations to the chain: https://ethereum.org/en/glossary/#transaction

By changing "interaction" to "transaction" we could probably just drop the rest:

```suggestion
Gas is required for any transaction that involves a smart contract.
```